target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -201,6 +201,9 @@ ramips_setup_interfaces()
 	wl-330n)
 		ucidef_set_interface_lan "eth0.1"
 		;;
+	mr200)
+		ucidef_set_interfaces_lan_wan "eth0" "usb0"
+		;;
```

Comment:
Please add a switch config here for the board. You are relying on settings done in [target/linux/ramips/base-files/lib/preinit/07_set_preinit_iface_ramips](https://github.com/lede-project/source/blob/40bfe9f0bdb4750a77140d1e402e9cf37326c38f/target/linux/ramips/base-files/lib/preinit/07_set_preinit_iface_ramips).

As soon as these settings are changed for some reason, networking for your board might not work anymore. You can set your wan interface using ```ucidef_set_interfaces_wan``` when having the switch config.